### ISSUE TITLE
[BUG] - Regression on displaying index that have a composite id field

### ISSUE THREAD

**Issue by maurin-at-homiwoo** [cars10/elasticvue#219]
Version 1.0.4 - Chrome Extension

**Description**
No entries are displayed in the Search tab for index that have a composite id field

Regression for the issue https://github.com/cars10/elasticvue/issues/142 but with a different stacktrace : 

SearchResultsTable.ts:68 TypeError: Cannot convert object to primitive value
    at f (table-row-selection.js:52:25)
    at Y (QTable.js:384:20)
    at QTable.js:466:29
    at Proxy.map (<anonymous>)
    at re (QTable.js:465:38)
    at H (QTable.js:327:9)
    at Proxy.<anonymous> (QTable.js:1038:9)
    at Fs (runtime-core.esm-bundler.js:874:16)
    at Qr.U [as fn] (runtime-core.esm-bundler.js:6089:26)
    at Qr.run (reactivity.esm-bundler.js:177:19)

**Steps To Reproduce**
List the steps to reproduce your problem:
1. Open elasticvue
2. Create a new index that has a composite id field with the following bulk request : 
```{ "index" : { "_index" : "test_id", "_id" : "1" } }
{ "id" : { "blip": "1", "blap": "2"}, "blop": "Blup" }
{ "index" : { "_index" : "test_id", "_id" : "2" } }
{ "id" : { "blip": "12", "blap": "22"}, "blop": "Blup3" }
{ "index" : { "_index" : "test_id", "_id" : "3" } }
{ "id" : { "blip": "13", "blap": "23"}, "blop": "Blup3" }
```

4. Go to the search tab and try to display all rows of the test_id index

**Screenshots**
See screenshots of the issue 142 (see link above)

**Environment (please include the following information):**
* Elasticsearch version: 8.10.2
* Operating system: Ubuntu 22.04
* Browser + version: Chrome 122.0.6261.128 
* Elasticvue version: 1.0.4-stable
* How are you running elasticvue? (docker, browser extension, web app): Chrome Extensio

**Additional context**
Thanks in advance, let me know if I can help in any way ! 
And thanks a lot for all your work :)

**Comment by cars10** [comment]
Fixed in `1.0.5`, thanks for reporting!

**Comment by maurin-at-homiwoo** [comment]
Thanks a lot ! :+1: :100:
